assets/model_monitoring/components/src/feature_importance_metrics/compute_feature_importance.py

```diff
@@ -270,17 +273,25 @@ def run(args):
         log_time_and_message("Reading data in spark and converting to pandas")
         baseline_df = try_read_mltable_in_spark_with_error(args.baseline_data, "baseline_data")
 
+        categorical_features = get_categorical_cols_with_df_with_override(baseline_df,
+                                                                              args.override_numerical_features,
+                                                                              args.override_categorical_features)
         baseline_df = baseline_df.toPandas()
-        task_type = determine_task_type(args.task_type, args.target_column, baseline_df)
+        # lightgbm does not support features that can't be converted to non-bool, int, float features
+        # if these features can't be converted to an int. Therefore we have to more these as "category"
+        # types
+        categorical_features_lgbm = compute_lightgbm_unsupported_categorical_features(baseline_df,
```

Comment:
I'm confused with the naming here, is categroical_features_lgbm stores for supported or unsupported lgbm categorical features?
Does this method do some lightgbm specific logic to further filter out some categorical columns? If so, maybe rename to sth like `get_lightgbm_categrocial_features()`

Reply:
Done